x-pack/solutions/security/plugins/security_solution/public/entity_analytics/pages/entity_analytics_management_page.tsx

```diff
@@ -49,12 +51,13 @@ export const EntityAnalyticsManagementPage = () => {
   const [isLoading, setIsLoading] = useState(false);
   const { mutate: scheduleNowRiskEngine } = useScheduleNowRiskEngineMutation();
   const { addSuccess, addError } = useAppToasts();
+  const userCanRunEngine =
```

Comment:
Should we have a common function somewhere for this which uses `TO_RUN_RISK_ENGINE_REQUIRED_ES_CLUSTER_PRIVILEGES` ?